obal/data/modules/check_koji_build.py

```diff
@@ -23,13 +23,24 @@ def main():
     package = module.params['package']
     koji_executable = module.params['koji_executable']
 
-    command = ['latest-build', '--quiet', tag, package]
+    if tag:
+        command = ['latest-build', '--quiet', tag, package]
+    else:
+        command = ['buildinfo', nevr]
 
     try:
         build = koji(command, koji_executable)
-        build = build.split(' ')[0]
 
-        module.exit_json(changed=False, tagged_version=build, exists=(nevr == build))
+        if tag:
+            build = build.split(' ')[0]
+
+            module.exit_json(changed=False, tagged_version=build, exists=(nevr == build))
+        else:
+            output = koji(command, koji_executable)
```

Comment:
this executes `koji(command)` a second time (the first one in line 32).

I'd argue for better readability to fold the `if/else` from lines 26-29 with the one here too